src/lib/index.js

```diff
@@ -0,0 +1,5 @@
+var convert = require('./workers/conversion').convert;
+
+module.exports = {
+    convert: convert
```

Comment:
Maybe "{ convert }" could be simpler than "{ convert: convert }" if the external name is the same with the internal name.

Reply:
Just curious why is this file named index.js?

Reply:
> Just curious why is this file named index.js?

From my perspective, the index.js filename is a convention as a function hub of a lib. And there is a syntactic sugar that `require('@/package/lib')` is equal to `require('@/package/lib/index.js')`.

Reply:
Thanks, I will change it "{ convert }".
I think, although the filename 'index.js' can be named arbitrarily, it generally defaults to index.js in node.js, which can be seen as the index of the exported variables and functions.

Reply:
Got it. Thanks!